05-aks-cluster.md

```diff
@@ -23,6 +23,24 @@ Now that the [hub-spoke network is provisioned](./04-networking.md), the next st
    RESOURCEID_VNET_CLUSTERSPOKE=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0008 --query properties.outputs.clusterVnetResourceId.value -o tsv)
    ```
 
+1. Enable the Azure Key Vault Secrets Provider feature
```

Comment:
Typically we like to do preview feature enablement like this in the pre-req section of the docs. We have a step that comes and goes if we are using a preview feature. Check the file history for the style of that. (Typical we link to the instructions instead of duplicating them)

Reply:
The format is the following.

```markdown
1. While the following feature(s) are still in _preview_, please enable them in your subscription.
   1. [Register the Azure Policy for AKS preview feature - `AKS-AzureKeyVaultSecretsProvider`](https://docs.microsoft.com/azure/...).
```

And we inject that after the Azure CLI install step